tests/plugins/image2/resizerwithreadonly.js

```diff
@@ -42,7 +41,31 @@
 					listener.removeListener();
 
 					resume( function() {
```

Comment:
Your tests may be a bit simplified by reusing `editor.setData` the second argument which is a callback called when data is ready (basically the same logic you proposing although hidden by nice API). More info at https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_editor.html#method-setData

```javascript
editor.setData( '<img src="_assets/foo.png" alt="" />', function() {
	resume( function() {
		var resizer = editor.editable().findOne( '.cke_widget_image .cke_image_resizer' );
		assert.isTrue( Boolean( resizer ), 'Resizer should be enabled'  );
	} );
} );

wait();
```